site/docs/pages/pay/pay.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: <Pay /> · OnchainKit
+description: One-click checkout for onchain commerce
+---
+
+import { Pay } from '@coinbase/onchainkit/pay';
+
+# `<Pay />`
+
+The `Pay` component provides a one-click checkout experience for onchain commerce. 
+
+Our all-in-one solution simplifies payment processing for onchain developers, removing complex integrations, high fees, and onboarding friction. Whether you're selling digital goods, services, or in-game items, this tool is for you.
+
+## Features
+- **Plug-and-Play Integration:** Add our `Pay` button with just a few lines of code. No backend required.
+- **Seamless Onboarding:** Support Passkey wallets to eliminate onboarding drop-offs.
+- **Real-time Merchant Tooling:** Get instant payment tracking, analytics, and reporting.
+
+## Quick start
+
+Before using the `Pay` component, ensure you've completed all [Getting Started steps](/getting-started).
+
+If you're starting a new project, we recommend using [Wagmi](https://wagmi.sh/) to scaffold your project following our [Getting Started](/getting-started) steps.
+
+If you're adding the component to an existing project, simply install OnchainKit (`npm install @coinbase/onchainkit@latest`). Wrap the `<OnchainKitProvider />` around your app following steps in [Getting Started](/getting-started).
+
+## Walkthrough
+
+::::steps
+
+## Sign up for a Coinbase Commerce account
+Head to [Coinbase Commerce](https://beta.commerce.coinbase.com/) and sign up. This is where you’ll manage transactions, view reports, and configure payments.
+
+## Create a product and copy the `productId`
+In the Coinbase Commerce dashboard, create a new product and copy the `productId`.
```

Comment:
Maybe include a screenshot of what this looks like on the Commerce dashboard?

Similar to the `Get Your Public API Key` section in the `Getting Started` guide:
![Screenshot 2024-10-08 at 2 16 01 PM](https://github.com/user-attachments/assets/224d2fe9-4e64-4fd1-a1e3-affe676acc16)

Reply:
> I see this twoslash error under the Fund example [here](https://onchainkit-ge7rw4y16-coinbase-vercel.vercel.app/):

Fixed with the latest! 

> Attaching image of Commerce

Agreed! I will add that once the Commerce product updates. Now the page is incorrect, so needs to add those in after the update, but don't want to unblock sending this page to customers.